### ISSUE TITLE
converting html with cdn styling issue

### ISSUE THREAD

**Issue by derryhandoyo** [wkhtmltopdf/wkhtmltopdf#4136]
The exit status code '1' says something went wrong: stderr: "Loading pages (1/6) [> ] 0% [======> ] 10% [=======> ] 13% Error: Failed to load https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css, with network status code 3 and http status code 0 - Host not found Error: Failed to load https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css, with network status code 3 and http status code 0 - Host not found 

is there a solution with these error?
it's happen in a connected to internet pc.. i can just comment out the styling and it can be converted to pdf but with bad styling..

**Comment by Tomsgu** [comment]
We can't help without any example. What do you mean by connected to internet pc? Are you trying to use cdn without an internet connection? What is your wkhtmltopdf version?

**Comment by derryhandoyo** [comment]
nope.. i access cdn within pc with an internet connection.. and here just a simple pdf export i got from some crud generator.. if i comment out the cdn css it export datatables succesfully.. but if i use the bootstrap cdn i got those error..
https://gist.github.com/derryhandoyo/872d3f174c9f64397ee0aaa3d0fc09c4

**Comment by PhilterPaper** [comment]
Perhaps the CDN is a bit slow responding, due to load or network issues, and wkHTMLtoPDF is timing out? Try a flag to control how long it waits for content to load or Javascript to run (there are several) and see if you can get the problem to go away with longer waits. I'm assuming it's permitted to load a local HTML file, but get resources from the Web.

**Comment by derryhandoyo** [comment]
i try it with simple content and yet it still return same error.. and btw it took less than a seconds to load..so i think there is no issue with the load of network..
here the full error that i got

`
The exit status code '1' says something went wrong: stderr: "Loading pages (1/6) [> ] 0% [======> ] 10% [========> ] 14% Error: Failed to load https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css, with network status code 3 and http status code 0 - Host not found Error: Failed to load https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css, with network status code 3 and http status code 0 - Host not found [============================================================] 100% Counting pages (2/6) [============================================================] Object 1 of 1 Resolving links (4/6) [============================================================] Object 1 of 1 Loading headers and footers (5/6) Printing pages (6/6) [> ] Preparing [============================================================] Page 1 of 1 Done Exit with code 1 due to network error: HostNotFoundError " stdout: "" comm`and:

**Comment by PhilterPaper** [comment]
I can't find a listing for "network status code 3", so I don't know what that is. It _does_ then say "Host not found". That URL seems to be fine (and it's not modified or redirected with a 301 status), but my browser says it's a 304 status code ("not modified"), so maybe wkHTMLtoPDF isn't correctly handling 3xx codes? I've seen suggestions of that in 301 or 302 redirects apparently not being properly handled. Otherwise, my best guess would be that a firewall or something near your end is blocking that site for some reason. Are you behind a firewall?

**Comment by derryhandoyo** [comment]
hmm have tried both with firewall on and off and seems nothing different.. same error..
